website/events-listener/src/producer.ts

```diff
@@ -26,10 +25,11 @@ export class KafkaProducer {
     const admin = this.kafka.admin();
     try {
       await admin.connect();
-      log.info('Admin is connected');
+      logger.info(`${EVENTS_LISTENER} Kafka producer: Admin is connected`);
```

Comment:
EVENTS_LISTENER actually isn't necessary here. It's obvious. Kafka producer is more than enough

Reply:
The same thing in all cases of this file

Reply:
It might be a not good idea. Without the `EVENTS_LISTENER` in `terminal` will little bit hurt to recognize from where the logger was called. For that, I include the service `name`.

Or do you @osipov-mit  think it will be better without `service name`?